src/pages/player/player.jsx

```diff
@@ -96,7 +96,7 @@ module.exports = React.createClass({
           <FullWidthButton onClick={this.setDestination} off={this.state.params.mode !== 'link' || !this.state.selectedEl}>Set Destination</FullWidthButton>
         </Menu>
 
-        <Loading on={this.state.loading} />
```

Comment:
This is what's causing the failure right now (since it uses i18n, which is never initialized)